src/WinRT.Runtime/Projections/NotifyCollectionChangedEventHandler.cs

```diff
@@ -40,12 +40,13 @@ static unsafe NotifyCollectionChangedEventHandler()
             };
             var nativeVftbl = ComWrappersSupport.AllocateVtableMemory(typeof(NotifyCollectionChangedEventHandler), sizeof(global::WinRT.Interop.IDelegateVftbl));
             *(global::WinRT.Interop.IDelegateVftbl*)nativeVftbl = AbiToProjectionVftable;
-            AbiToProjectionVftablePtr = nativeVftbl;
+            AbiToProjectionVftablePtr = nativeVftbl;
+            ComWrappersSupport.RegisterDelegateFactory(typeof(global::System.Collections.Specialized.NotifyCollectionChangedEventHandler), CreateRcw);
         }
 
         public static global::System.Delegate AbiInvokeDelegate { get; }
 
-        private static readonly Guid IID = new(0x8B0909DC, 0x2005, 0x5D93, 0xBF, 0x8A, 0x72, 0x5F, 0x01, 0x7B, 0xAA, 0x8D);
+        public static Guid IID { get; } = global::WinRT.Interop.IID.IID_NotifyCollectionChangedEventHandler;
```

Comment:
Same here. If we return the RVA field directly the JIT can inline this across calls too:
```suggestion
        public static Guid IID => global::WinRT.Interop.IID.IID_NotifyCollectionChangedEventHandler;
```